polkadot/node/core/provisioner/src/lib.rs

```diff
@@ -421,13 +421,7 @@ async fn send_inherent_data(
 		"Selected disputes"
 	);
 
-	// Only include bitfields on fresh leaves. On chain reversions, we want to make sure that
-	// there will be at least one block, which cannot get disputed, so the chain can make progress.
```

Comment:
how do we ensure that?

Reply:
By not including bitfields in the very first block after a reversion. If there are no bitfields, nothing can become included as of that block, hence it can not become a reversion target gain.

Reply:
Given that the already existing leaf we are reverting to already exists, the stale/fresh mechanism should not be necessary to achieve this as we should not get that leaf activated again, hence nothing in parachain consensus is supposed to do anything -> paras inherent should be empty/not existing.

Reply:
Nice! So it works already 👍